apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -130,6 +139,46 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
         setSelected('')
     }
 
+    const searchMoreSuggestions = useCallback(
+        async (value, skip) => {
+            if (isAllData) return
+
+            setLoading(true)
+
+            const data = await search(
+                client,
+                getSearchTextValue(value),
+                null,
+                10,
+                skip
+            )
+
+            setLoading(false)
+
+            if (data.length > 0) {
+                setData(prevData => [...prevData, ...data])
+            } else {
+                setIsAllData(true)
+            }
+        },
+        [],
+    )
+
+    const throttledSearchMore = useMemo(
```

Comment:
looks like some of the functions here could be made reusable by moving to useInfiniteScroll hook

Reply:
Seems like yes, but I guess it should be in another task, if current is having increased priority, isn't it?